main.c

```diff
@@ -15,9 +15,51 @@ const char *SORT_NAMES[] = {"Merge Sort", "Quick Sort", "Bucket Sort"};
 // and return the number of inversions. Uncomment case 1 in InversionsHelper
 // function to test code. Compare your algorithms output to the correct # of
 // inversions
-//  int MergeSort(int arr[]) {
+ int MergeSort(int arr[], int start, int end) {
+   
+    int inversion_MergeSort = 0;
 
-// }
+    if (start < end){
+
+      int mid = start+(end-start)/2;
+
+      int pointer_left = start;
+      int pointer_right = mid+1;
+
+      inversion_MergeSort += MergeSortAndCount(a, pointer_left, mid);
```

Comment:
MergeSortAndCount(...) is not recognized. Perhaps it is incorrectly named or the function is missing or out of scope?

Reply:
Yes, we use different name for the function, and I forgot to change it in the recursions when inserted.